r2t2/docstring_parser.py

```diff
@@ -44,10 +55,21 @@ def iter_extract_docstring_from_lines(
 def iter_extract_docstring_from_file(
     path: Union[str, Path]
 ) -> Iterable[CodeDocumentComment]:
-    return iter_extract_docstring_from_text(
-        Path(path).read_text(),
-        filename=str(path)
-    )
+    path = Path(path)
+    txt = path.read_text()
```

Comment:
The error on Windows seems to be a text decoding problem, and it's occurring here. Perhaps specifying the `encoding=` parameter to [Path.read_text()](https://docs.python.org/3/library/pathlib.html#pathlib.Path.read_text) would solve the problem?

Reply:
Looks like specifying it as `utf-8` fixed the issue, thanks!